tests/ui/test_ui_tools.py

```diff
@@ -1043,6 +1043,7 @@ def test_init(self, right_col_view):
         assert right_col_view.view.user_search == right_col_view.user_search
         self.thread.Lock.assert_called_with()
         assert right_col_view.search_lock == self.thread.Lock()
+        assert right_col_view.focus_index_before_search == 0
```

Comment:
minor: The commit title is not in the imperative tense. :)

Reply:
Oh sry I'll change that